upgrade/UPGRADE-v9.0.0-dev.md

```diff
@@ -567,6 +545,7 @@ There you can find links to upgrade notes for other versions too.
                 +   {% set address = user.customer.billingAddress %}
                 ```
         - method `Shopsys\FrameworkBundle\Model\Customer\UserFacade::createCustomerWithBillingAddress()` was extracted to new class `Shopsys\FrameworkBundle\Model\Customer\CustomerUserFacade`
+        - review all templates, controllers and form types and rename all occurrences of user to customerUser [see project-base diff](https://github.com/shopsys/project-base/commit/9a5b86f91204e6da20e7eeb2680cf3678483ddb5) for inspiration
 
 - add hover timeout to horizontal menu ([#1564](https://github.com/shopsys/shopsys/pull/1564))
     - you can skip this task if you have your custom design
```

Comment:
diff below sentence "move loader to hidden submenu" is written in old style JS 
`Shopsys.ajax({`

Reply:
fixed